packages/commerce-sdk-react/scripts/build-and-release-docs.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const sh = require('shelljs')
+const semver = require('semver')
+
+// this script is expected to run on release branch.
+const main = () => {
+    // get the version number from package json
+    const {stdout: currentVersion} = sh.exec(
```

Comment:
Is there any specific thoughts on simply requiring the package.json file directly like this:

 ```
 const packageJSON = require('./package.json')
 console.log('version: ', packageJSON.version
 ```
 
 Alternatively if you want to use 'sh.exec', this might be a more concise command to run:
 
 ```
const version = sh.exec('node -e "console.log(require('./package.json').version);"')
 ```